tests/syntax_tests/data/ppx/react/expected/externalWithCustomName.res.txt

```diff
@@ -8,7 +8,7 @@ module Foo = {
   external component: React.componentLike<props<int, string>, React.element> = "component"
 }
 
-let t = React.createElement(Foo.component, {a: 1, b: "1"})
+let t = React.createElement(Foo.component, {a: 1, b: {"1"}})
```

Comment:
Is this test output change ok/expected?

Reply:
Note that this is not a printing test, but shows the result of the jsx transform. So it aims to show the effect of the transformation.
That said, the braces `{` were present in the source, so it's likely that these are preserved better now because there are no `res.namedArgLoc` attributes around to confuse the printing logic.